### DISCUSSION TITLE
Adaptation TTS for the Belarusian language

### DISCUSSION THREAD

**Discussion by Kryuski** [snakers4/silero-models#213] [Q&A]
I'm trying to adapt the Ukrainian language TTS model v3_ua.pt to the Belarusian language. I had an issue with a pronunciation of the letter "**Ў**", which has no direct analogue in Ukrainian.
When I pass "**Ў**" as "**У**", it sounds too long.
I tried to shorten the duration of this letter using the SSML tag **prosody**:
`<speak>дагадзі<prosody rate="x-fast">у</prosody></speak>`
But in this case, the generator separates the letter from the rest of the word, inserting a pause as between adjacent words.

Can you recommend another solution to the issue, or is it not solvable without training a separate Belarusian-language TTS model?

**Comment by snakers4** [comment]
Hi,

I believe that the parser does treat separate tags as separate words. In any case if we wanted to add this language, we would do it.

**Reply by Kryuski** [reply]
I'm afraid that the creation of a separate Belarusian-language TTS model will not pay for itself. But for non-commercial use, your Ukrainian TTS is quite suitable if it receives a Belarusian text pre-processed in a special way as input. The only problem is the missing analogue of the letter Ў. In some cases I was able to find the correct transcriptions, but not in all.
